gilbreth_perception/README.md

```diff
@@ -1,5 +1,14 @@
-1.Save gilbreth_perception folder under your-catkin-workspace/src.
+For using Correspondence Grouping method in object recognition:
+1.Run command "roslaunch gilbreth_perception gilbreth_perception.launch cnn:=false".
 
-2.Under your-catkin-workspace run command "catkin_make".
-
-3.Run command "roslaunch gilbreth_perception gilbreth_perception.launch".
+For using convolutional neural network method in object recognition:
+1. Make sure your computer has Nvidia GPU;
+2. Install voxnet at https://github.com/dimatura/voxnet;
+- Install all dependent packages as instructed on the webpage
+- Remember to add voxnet path into your PYTHONPATH by command:
+  ‘export PYTHONPATH=$PYTHONPATH:you-path-to-voxnet/voxnet’
+- Verify voxnet has been installed successfully by ‘import voxnet’ in python, make sure no error occurs
```

Comment:
The server I am using works with theano verson 1.0 and the lasagne by following installation instruction here: https://github.com/Lasagne/Lasagne.
I will add your case in the readme file, in case others have the same issue.